INSTALL.md

```diff
@@ -1,3 +1,30 @@
 # Installation instructions
 
-Detailed instructions on how to install, configure, and get the project running.
+Detailed instructions on how to clone and run NextGen_Regionalization are provided below. The workflow requires the following Python libraries:
+  * `yaml` (`pip install pyyaml`)
+  * `scikit-learn` 
+  * `scikit-learn-extra`
+  * `matplotlib`
+  * `geopandas` 
+  * `hdbscan`
+  * `numba` 
+
+## Clone the repository
+```
+git clone https://github.com/NOAA-OWP/NextGen_Regionalization
+cd NextGen_Regionalization
+```
+
+## Run the Example Regionalization Workflow ([presented at AMS 2024](https://docs.google.com/presentation/d/1xkYs-Hs3_cmIheLZ1Di7Dy3vWaiL3jmx/edit?usp=sharing&ouid=117267696082803250432&rtpof=true&sd=true))
+0. Activate the virtual python environment with all dependencies installed
+
+1. Go to the example directory
+```
+cd NWMv4/ams2024 
+```
+
+2. Download the [hydrofabric data](https://www.lynker-spatial.com/#hydrofabric/v20.1/gpkg/) for HUC 12 (Texas-Gulf Region) to `NextGen_Regionalization/NWMv4/datasets/nextgen_12.gpkg`
```

Comment:
A future improvement could be to show how to download using an AWS api.

Reply:
Or perhaps specify the exact url, although I don't know whether this could change: https://lynker-spatial.s3-us-west-2.amazonaws.com/hydrofabric/v20.1/gpkg/nextgen_12.gpkg

Reply:
I'm confused on whether this should be in `NextGen_Regionalization/NWMv4/datasets` or `NextGen_Regionalization/NWMv4/data`. If it's the former, let's add in a "create a directory named 'datasets'" and we may want to make sure it's ignored in the .gitignore file.

Reply:
if I can make a suggestion. We usually divide the datasets into static and dynamic. Hydrofabric is a static set (for now) and generally should be separated from forcings or any assimilated data.